frontend/src/Login/Login.tsx

```diff
@@ -1,15 +1,26 @@
+import { Box } from '@mui/material';
 import { useTranslation } from 'react-i18next';
-
+import './Login.css';
 
 function Login() {
-  const {t, i18n} = useTranslation();
+  const {t} = useTranslation();
```

Comment:
It' better to use `{ keyPrefix: "login" }` parameter, same as I commented in User's page review